### PR TITLE
Release v1.2.20.0

### PR DESCRIPTION
**Pull request by iulianpascalau**
Activation epoch for the rewards checkpoint optimization is epoch 427 which should happen on 2021.09.30 ~14:45 UTC
Activation epoch for Arwen v1.4 & other fixes is epoch 432 which should happen on 2021.10.05 ~14:45 UTC

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -11,68 +11,11 @@
    # available in local disk
    StartInEpochEnabled = true
 
-   # SCDeployEnableEpoch represents the epoch when the deployment of smart contracts will be enabled
-   SCDeployEnableEpoch = 180
+   # ChainID identifies the blockChain
```
Comment (bogdan-rosianu) [inline config.toml:14]: could have been a more explanatory comment

**Review by bogdan-rosianu — DISMISSED** [review]

File: config.toml (hunk 1 above)
Comment (iulianpascalau) [reply config.toml:14]: will be done on elrond-go master/development

**Review by raduchis — APPROVED** [review]

File: systemSmartContractsConfig.toml [hunk 2]
```diff
@@ -20,25 +15,26 @@
...
-    EnabledEpoch = 272
 
 [GovernanceSystemSCConfig]
-    ProposalCost = "5000000000000000000" #5 eGLD
-    NumNodes     = 500
-    MinQuorum    = 400
-    MinPassThreshold = 300
-    MinVetoThreshold = 50
-    EnabledEpoch = 1000000
+    FirstWhitelistedAddress = "erd1agy0x6qrfmhgvyjrurwxrhf6a2xgcxmgmx5l4dw2w6xpa223zr5s04hkw4"
```
Comment (sasurobert) [inline systemSmartContractsConfig.toml:20]: we coud have used the multisig address for this: erd1qqqqqqqqqqqqqpgq0gq09608js3gje72v0yzc56k6glc852gl94sgtahk2

**Review by sasurobert — APPROVED** [review]

File: systemSmartContractsConfig.toml (hunk 2 above)
Comment (iulianpascalau) [reply systemSmartContractsConfig.toml:20]: right :facepalm:

Comment (iulianpascalau) [reply systemSmartContractsConfig.toml]: done

**Review by sasurobert — DISMISSED** [review]

**Review by bogdan-rosianu — DISMISSED** [review]

**Review by raduchis — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Pull request merged**